subsys/net/lib/sockets/sockets.c

```diff
@@ -851,6 +853,36 @@ static int sock_get_pkt_src_addr(struct net_pkt *pkt,
 	return ret;
 }
 
+void net_socket_update_tc_rx_time(struct net_pkt *pkt, uint32_t end_tick)
+{
+	net_pkt_set_rx_stats_tick(pkt, end_tick);
+
+	net_stats_update_tc_rx_time(net_pkt_iface(pkt),
+				    net_pkt_priority(pkt),
+				    net_pkt_timestamp(pkt)->nanosecond,
+				    end_tick);
+
+	if (IS_ENABLED(CONFIG_NET_PKT_TXTIME_STATS_DETAIL)) {
```

Comment:
Should be `CONFIG_NET_PKT_RXTIME_STATS_DETAIL`

Reply:
> Should be `CONFIG_NET_PKT_RXTIME_STATS_DETAIL`

Indeed, good catch